vignettes/detailed-canvas-export-info.Rmd

```diff
@@ -0,0 +1,105 @@
+
+Assignment Name - the Name of the Assignment that was submitted late.
+
+Points Possible - This is the maximum number of points the assignment is worth. 
+
+Due Date - This is the Date the assignment was due. Unfortunately, it only provides the due date and does not provide the time the assignment was due. This is an issue when calculating lateness (to be covered next). 
+
+Submitted Date - The date the assignment was submitted including the exact time of submission with time zone. 
+
+Graded Date - The date and time when the assignment was graded including the time zone. 
+
+Posted Date - The date and time when the assignment grade was posted to students including the time zone. 
+
+### Notes:
+
+With some data wrangling, this csv can be aggregated with the grades csv to add lateness and submission time information, which will make the csv more similar to a gradescope csv. 
+
+However, the lack of a due time information makes it impossible to calculate lateness with an error upper bound of less than 12 hours. My current implementation assumes the due time is at 12:00 AM on the due date (the least generous to students) so assignments may be marked as up to 24 hours later than they really were. For example, our example class often had quizzes due at 9:30 AM (the beginning of lecture). In one case, a student submitted at 9:46 AM, and thus would be marked as 9 hours 46 minutes late by this assumption rather than 16 minutes late. Note that since only assignments that were submitted late are included in this csv, over-assumption of lateness only applies to assignments that we submitted late. For example, a quiz submitted at 9:26 AM would not be in the late csv and thus would not be given any lateness in our processing. 
+
+As a solution, we command that lateness not be available to users uploading information from Canvas since Canvas offers limited abilities to apply simple late policies internally. When applied, a student’s score in the grades csv would be after lateness was applied. 
+
+It appears that it may be possible to obtain a spreadsheet of assignment due times using Canvas API (however this may be complicated). This is worth looking into during the future so that Gradebook can accommodate applying lateness to Canvas exported data. 
+
+## Roster CSV:
+
+This CSV file contains information providing student email addresses in case we want to add them into the grading dataframe. Each row corresponds to a particular student in the course. 
+
+It contains the following columns:
+
+Student Name - This is in a different format than the other two CSVs. It is in the format “First Last” as if someone wrote their name in a typical fashion. 
+
+Student ID - This is the Canvas internal ID for each student. 
+
+Student SIS ID - This is the CalCentral id. It also contains examples of “UID:  xxxxxx” with seemingly greater frequency than the other CSVs. 
+
+Email - This column contains the Student’s email address!
+
+Section Name - like in other CSVs.
```

Comment:
I'll do a PR to Zach's repo since it's easier to edit vignettes that way than through Github review